sdks/python/apache_beam/runners/common.py

```diff
@@ -402,13 +403,17 @@ def finish(self):
   def _reraise_augmented(self, exn):
     if getattr(exn, '_tagged_with_step', False) or not self.step_name:
       raise
-    args = exn.args
-    if args and isinstance(args[0], str):
-      args = (args[0] + " [while running '%s']" % self.step_name,) + args[1:]
-      # Poor man's exception chaining.
```

Comment:
Can you retain a comment that this function is intended to implement exception chaining?